transport_analysis/tests/test_viscosity.py

```diff
@@ -0,0 +1,196 @@
+import pytest
+from numpy.testing import assert_allclose
+
+from transport_analysis.viscosity import (
+    ViscosityHelfand as VH,
+)
+import MDAnalysis as mda
+from MDAnalysis.units import constants
+from MDAnalysis.transformations import set_dimensions
+import numpy as np
+
+from MDAnalysis.exceptions import NoDataError
+from MDAnalysisTests.datafiles import PRM_NCBOX, TRJ_NCBOX, PSF, DCD
+
+
+@pytest.fixture(scope="module")
+def u():
+    return mda.Universe(PRM_NCBOX, TRJ_NCBOX)
+
+
+@pytest.fixture(scope="module")
+def ag(u):
+    return u.select_atoms("name O and resname WAT and resid 1-10")
+
+
+@pytest.fixture(scope="module")
+def u_no_vels():
+    return mda.Universe(PSF, DCD)
+
+
+@pytest.fixture(scope="module")
+def ag_no_vels(u_no_vels):
+    return u_no_vels.select_atoms("backbone and name CA and resid 1-10")
+
+
+@pytest.fixture(scope="module")
+def NSTEP():
+    nstep = 5001
+    return nstep
+
+
+@pytest.fixture(scope="module")
+def visc_helfand(ag):
+    vh_t = VH(ag, fft=False)
```

Comment:
Parametrize over fft=True and fft=False

Reply:
I'll remove FFT for now then and raise an issue.

Reply:
:+1: